src/routes/ChatForm.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import { FBrealtime } from '../fbase';
+import { ref, set } from '@firebase/database';
+import styled from 'styled-components';
+
+type ChatFormProps = {
+  userObj: any;
+  chatRoomID: string[];
+};
+
+const ChatForm = ({ userObj, chatRoomID }: ChatFormProps) => {
+  const [message, setMessage] = useState('');
+  const now = new Date();
+
+  const handleSubmitForm = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    //공백 메시지 입력 방지
+    if (message === '') {
```

Comment:
```suggestion
    if (!message) {
```
이렇게 하면 더 가독성이 좋을 것 같아요!